### PR TITLE
change(github): Add release process (ESPTOOL-871)

### PR DESCRIPTION
**Pull request by dobairoland**
- Stub names are changed to be the same as in https://github.com/esp-rs/esp-flasher-stub/releases.
- DangerJS will start to work once this is merged (according to Tomas S.). I've tested this in my fork. See:
image
- I've tagged master as v1.0.0. The empty `feat` commit will trigger the initial release (v1.1.0). I plan to do this in the next PR in order to avoid to re-tag the new release. Here is a release published without modification: https://github.com/dobairoland/esptool-legacy-flasher-stub/releases/tag/v0.1.0

### CODE REVIEW

**Comment by dobairoland** [comment]
@radimkarnis @peterdragun @DNedic PTAL

**Comment by DNedic** [comment]
LGTM!

File: .codespellrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[codespell]
+ignore-words-list = bloc,ser,dout,exten
```
Comment (peterdragun) [inline .codespellrc:2]: I think we should be able to drop this ignore list. I don't think it will be relevant in this repository.

**Review by peterdragun — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -10,6 +16,28 @@ Please report issues in [the esptool repository](https://github.com/espressif/es
...
+```bash
+pip install commitizen
+git fetch
+git checkout -b update/release_v1.1.0
+git reset --hard origin/master
+cz bump
+git push -u
+git push --tags
+```
+Create a pull request and edit the automatically created draft [release note](https://github.com/espressif/esptool-legacy-flasher-stub/releases).
```
Comment (radimkarnis) [inline README.md:39]: [release notes]

**Comment by radimkarnis** [comment]
LGTM! Left a small suggestion. Thanks.

**Comment by dobairoland** [comment]
@radimkarnis @DNedic @peterdragun Thank you for the review. All suggestions were accepted.

**Pull request merged**